install/syndesis-privileged.yml

```diff
@@ -469,7 +471,10 @@ objects:
           "logging": false
         },
         "branding": {
-          "logoDarkBg": "assets/images/syndesis_logo_icon_darkbkg.png",
+          "logoWhiteBg": "assets/images/syndesis-logo-svg-white.svg",
+          "logoDarkBg": "assets/images/syndesis-logo-svg-white.svg",
+          "iconWhiteBg": "assets/images/glasses_logo_square.png",
+          "iconDarkBg": "assets/images/glasses_logo_square.png",
```

Comment:
Not how this comes in here.

Reply:
I put that back after merge with master. Evidently it wasn't generated like you said using the install.../run script. 
I didn't want to rock any boats, so I put it back. :)

Reply:
Regenerated syndesis.yml using `/install/generator/run.sh`